src/shogun/classifier/svm/LibLinear.cpp

```diff
@@ -222,7 +224,7 @@ bool LibLinear::train_machine(std::shared_ptr<Features> data)
 		error("Error: unknown solver_type");
 		break;
 	}
-
+	
```

Comment:
please try to watch for these unnecessary whitespace changes...

Reply:
Oh my bad okay, will do from now on.

Reply:
the whitespace is still there :<